packages/base/room.gts

```diff
@@ -561,6 +612,20 @@ interface RoomCreateEvent extends RoomStateEvent {
   };
 }
 
+interface RoomJoinRules extends RoomStateEvent {
+  type: 'm.room.join_rules';
+  content: {
+    // TODO
+  };
+}
+
+interface RoomPowerLevels extends RoomStateEvent {
+  type: 'm.room.power_levels';
+  content: {
+    // TODO
+  };
+}
+
```

Comment:
Are these needed because they occur in tests or something and need to be skipped over?

Reply:
The intent here is to provide stronger types that what matrix provides so we can keep our events well typed. But yeah, this particular event only happens in a test, and in that test we really don’t care about the deeper details for this event. The TODO here is for when we do care about the deeper details of this event (if that ever happens)